.python_package/README.md

```diff
@@ -0,0 +1,73 @@
+# Python package for GridTools headers and CMake files
+
+## Usage
+
+Use the following when compiling C++ code agains GridTools programatically from Python.
+Either by calling a compiler directly or by generating a CMake project and calling CMake on it.
+
+```python
+import gridtools_cpp
+include_dir = gridtools_cpp.get_include_dir()   # header files can be found here
+cmake_dir = gridtools_cpp.get_cmake_dir()       # cmake files can be found here
+```
+
+## Development
+
+In order to be able to work on this package, it is necessary to run a preparation step.
+This will generate the `setup.cfg` file from `setup.cfg.in` and install the gridtools header distribution into the package data.
+It will read the version number from the top-level `version.txt` and copy the `LICENSE` file to where packaging tools can find it.
+
+All of this requires `nox`, the preparation step runs in an isolated environment and installs additional requirements `cmake` and `ninja` at runtime.
+
+```bash
+# pip install nox
+# nox -s prepare
```

Comment:
```suggestion
pip install nox
nox -s prepare
```

Reply:
why `#`? it makes it harder to copy paste.

Reply:
True